cmd/server/handlers/handlers.go

```diff
@@ -0,0 +1,99 @@
+package handlers
+
+import (
+	"evgen3000/go-musthave-metrics-tpl.git/cmd/server/storage"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/go-chi/chi/v5"
+)
+
+const (
+	MetricTypeCounter = "counter"
+	MetricTypeGauge   = "gauge"
+)
+
+type Handler struct {
+	storage *storage.MemStorage
+}
+
+func NewHandler(storage *storage.MemStorage) *Handler {
+	return &Handler{storage}
+}
+
+func (h *Handler) HomeHandler(rw http.ResponseWriter, _ *http.Request) {
+	var body strings.Builder
+	body.WriteString("<h4>Gauges</h4>")
```

Comment:
[SUGGESTION] It would be better if you used `template`
https://pkg.go.dev/text/template